common-lib/test/cmr/common/test/api/context.clj

```diff
@@ -0,0 +1,26 @@
+(ns cmr.common.test.api.context
+  "This tests capabilities of the API context utilities."
+  (:require
+   [clojure.test :refer :all]
+   [cmr.common.util :as util]
+   [cmr.common.api.context :as context]))
+
+(deftest request-context-test
```

Comment:
Can you write a comment or something to show what use cases these are testing?

Reply:
Added 'testing' map around tests to give some info